pkg/oidc/verify_test.go

```diff
@@ -0,0 +1,202 @@
+package oidc
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+	"github.com/coreos/go-oidc"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/square/go-jose.v2"
+)
+
+var _ = Describe("Verify", func() {
+	var (
+		ctx context.Context
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+	})
```

Comment:
I don't think this is necessary, is the context being overriden anywhere?

The following is probably sufficient in the case of a context
```suggestion
	ctx := context.Background()
```